subprojects/luajit/meson.build

```diff
@@ -37,5 +37,11 @@ incdir = include_directories('include')
 
 subdir('src')
 
+luajit_link_args = []
+if host_machine.system() == 'darwin'
+	luajit_link_args = ['pagezero_size', '10000', '-image_base', '100000000']
```

Comment:
This is triggering build errors, I guess because you forgot a hyphen? But I also consider this the wrong fix – we should just be using upstream LuaJIT. This is relatively easy to do outside of handling the patches needed for Windows, and since a proper upstream solution for that is not incoming I'll just handle it with custom targets. See https://github.com/CoffeeFlux/Aegisub/commits/luajit-upstream

Reply:
Yes, forgot a hyphen, sorry. Working on two different computers.